ops-bedrock/docker-compose.yml

```diff
@@ -67,7 +67,8 @@ services:
       - "8060:6060"
     volumes:
       - "l2_data:/db"
-      - "${PWD}/../.devnet:/config"
```

Comment:
Will this still work for our AWS deployments? There were problems when trying to map individual files and we had to mount whole directories.
Another way to solve the problem is to modify the Python script so that instead of calling openssl to create the secret file, it copies the existing one from ops-bedrock.

Reply:
Yes, the AWS deployment is built from our erigon repo.

Reply:
The openssl command doesn't exist in the `bedrock-devnet/devnet/__init__.py`.